src/azure-cli/azure/cli/command_modules/cosmosdb/aaz/latest/cosmosdb/postgres/configuration/node/_update.py

```diff
@@ -0,0 +1,292 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "cosmosdb postgres configuration node update",
+)
+class Update(AAZCommand):
+    """Updates configuration of worker nodes in a cluster
+    """
+
+    _aaz_info = {
+        "version": "2022-11-08",
+        "resources": [
+            ["mgmt-plane", "/subscriptions/{}/resourcegroups/{}/providers/microsoft.dbforpostgresql/servergroupsv2/{}/nodeconfigurations/{}", "2022-11-08"],
+        ]
+    }
+
+    AZ_SUPPORT_NO_WAIT = True
+
+    def _handler(self, command_args):
+        super()._handler(command_args)
+        return self.build_lro_poller(self._execute_operations, self._output)
+
+    _args_schema = None
+
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        if cls._args_schema is not None:
+            return cls._args_schema
+        cls._args_schema = super()._build_arguments_schema(*args, **kwargs)
+
+        # define Arg Group ""
+
+        _args_schema = cls._args_schema
+        _args_schema.cluster_name = AAZStrArg(
+            options=["--cluster-name"],
+            help="The name of the cluster.",
+            required=True,
+            id_part="name",
+            fmt=AAZStrArgFormat(
+                pattern="^(?![0-9]+$)(?!-)[a-z0-9-]{3,40}(?<!-)$",
+                max_length=40,
+                min_length=3,
+            ),
+        )
+        _args_schema.configuration_name = AAZStrArg(
+            options=["-n", "--name", "--configuration-name"],
+            help="The name of the cluster configuration.",
+            required=True,
+            id_part="child_name_1",
+            fmt=AAZStrArgFormat(
+                pattern="^[-\w\._]+$",
+                max_length=90,
+                min_length=1,
+            ),
+        )
+        _args_schema.resource_group = AAZResourceGroupNameArg(
+            required=True,
+        )
+
+        # define Arg Group "Properties"
+
+        _args_schema = cls._args_schema
+        _args_schema.value = AAZStrArg(
+            options=["--value"],
+            arg_group="Properties",
+            help="Value of the configuration.",
```

Comment:
What's the value can be? Please add more description in its help.

Reply:
Value depends on the specific configuration, it could be "on"/"off", or a numeric value so we can't go in detail on the values.